be/src/storage/metadata_util.cpp

```diff
@@ -256,15 +260,47 @@ Status convert_t_schema_to_pb_schema(const TTabletSchema& tablet_schema, uint32_
 
         has_bf_columns |= column->is_bf_column();
 
-        if (tablet_schema.__isset.indexes) {
-            for (auto& index : tablet_schema.indexes) {
-                if (index.index_type == TIndexType::type::BITMAP) {
-                    DCHECK_EQ(index.columns.size(), 1);
-                    if (boost::iequals(tcolumn.column_name, index.columns[0])) {
-                        column->set_has_bitmap_index(true);
-                        break;
-                    }
+        column_map.emplace(boost::to_lower_copy(column->name()), column);
+    }
+
+    if (tablet_schema.__isset.indexes) {
+        for (auto& index : tablet_schema.indexes) {
+            TabletIndexPB* index_pb = schema->add_table_indices();
+            index_pb->set_index_id(index.index_id);
+            index_pb->set_index_name(index.index_name);
+            if (index.index_type == TIndexType::type::BITMAP) {
+                RETURN_IF(index.columns.size() != 1,
+                          Status::Cancelled("BITMAP index " + index.index_name +
+                                            " do not support to build with more than one column"));
+
+                index_pb->set_index_type(IndexType::BITMAP);
+                const auto& mit = column_map.find(boost::to_lower_copy(index.columns[0]));
+                RETURN_IF(mit == column_map.end(), Status::Cancelled("index column (" + index.columns[0] +
```

Comment:
plz remove this check here. It is possible that such index contain some column does not in tablet schema when we construct rollup